src/framework/shortcuts/data/shortcuts.xml

```diff
@@ -1226,4 +1231,545 @@
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>grace8after</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>grace16after</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>grace32after</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>appoggiatura</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>add-braces</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>add-brackets</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>composer-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>fingering-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>instrument-change-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>poet-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>nashville-number-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>part-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>roman-numeral-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>sticking-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>subtitle-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>title-text</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>add-remove-breaks</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>append-hbox</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>append-vbox</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>append-textframe</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>auto-beam</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>beam-32</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>beam-64</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>beam-mid</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>beam-start</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>copy-lyrics-to-clipboard</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>pitch-up-diatonic-alterations</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>pitch-down-diatonic-alterations</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>edit-style</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>interval-2</key>
+    <seq></seq>
+    </SC>
```

Comment:
Somehow I have missed a couple contexts here
I have noted this and will fix this alongside any further requested changes